README.md

```diff
@@ -174,9 +172,7 @@ At runtime, if a non-value type is placed inside a `Record` or `Tuple`, it is a
 
 At runtime, attempting to create a `Record` with a key that is not a `string` is a `TypeError`.
 
-At runtime, it is a `TypeError` to add a value to a `Record` or `Tuple` of any type except the following: `Record`, `Tuple`, `string`, `number`, `true`, `false`, and `null`.
-
-> `undefined` is not permitted, as that would violate the goal of lossless round-tripping with JSON.
+At runtime, it is a `TypeError` to add a value to a `Record` or `Tuple` of any type except the following: `Record`, `Tuple`, `string`, `number`, `symbol`, `true`, `false`, `undefined` and `null`.
```

Comment:
true and false aren’t types, do you mean “boolean”?

Reply:
Agreed, we meant `boolean`

Reply:
updated on master